.github/workflows/tag.yml

```diff
@@ -0,0 +1,24 @@
+name: RELEASE
```

Comment:
I feel like these should all have docs at the top. The YAML doc should state what the workflow/action is and when it is triggered.